examples/tfengine/folder_foundation.hcl

```diff
@@ -88,6 +89,31 @@ template "cicd" {
   }
 }
 
+template "groups" {
+  recipe_path = "{{$recipes}}/project.hcl"
+  output_path = "./groups"
+  data = {
+    project = {
```

Comment:
Do we need a block like this? Can the groups be included in devops field?

Reply:
I tried to make it isolated from the rest of the deployment for now so it can be used more generally. In `team.hcl`, I used the group template block to create groups that are used in other deployments (apart from devops, audit, cicd). But I do plan to integrate the creation of groups that are used in devops, audit and cicd in those deployments.